### PR TITLE
Katie dev

### PR DESCRIPTION
**Pull request by k-odonnell**

### CODE REVIEW

File: app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java [hunk 1]
```diff
@@ -252,14 +163,16 @@ public void onClick(View view) {
                 finish();
             }
         });
-        /*binding.settingsButton.setOnClickListener(new View.OnClickListener() {
+        binding.settingsButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                Intent intent = new Intent(getApplicationContext(), ?.class);
+                Intent intent = new Intent(getApplicationContext(), SettingsPage.class);
```
Comment (RomanWSU) [inline app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java:169]: Make sure that the settings page class matches up with the settings page that we have. And being able to click on different pages to get to the settings page.

File: app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java [hunk 2]
```diff
@@ -38,6 +41,16 @@ public class OutfitCreationActivity extends AppCompatActivity {
...
 
+    //clothing IDs
+    String topID = "";
+    String bottomsID = "";
+    String shoesID = "";
+    String headwearID = "";
+    String socksID = "";
+    String otherID = "";
+    String outfitName = "";
+    String outfit[] = new String[7];    //[0]: name [1]: top [2]: bottoms [3]: shoes [4]: headwear [5]: socks [6]: other
```
Comment (RomanWSU) [inline app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java:52]: Please make sure you don't comment out anything important so the files don't have errors.

File: app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java (hunk 1 above)
Comment (k-odonnell) [reply app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java:169]: double checked settings button functionality and name, everything matches up

File: app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java (hunk 2 above)
Comment (k-odonnell) [reply app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java:52]: got it

**Pull request merged**